Cluster/How to configure APIM for Service Fabric managed cluster.md

```diff
@@ -0,0 +1,695 @@
+        Name = 'apim'
+        VirtualNetwork = $virtualNetwork
+        AddressPrefix = '10.0.1.0/24'
+        NetworkSecurityGroup = $networkSecurityGroup
+    }
+
+    Add-AzVirtualNetworkSubnetConfig @sfmcSubnet
+    Add-AzVirtualNetworkSubnetConfig @apimSubnet
+    ```
+
+1. Then associate the subnets configuration to the virtual network with Set-AzVirtualNetwork.
+
+    ```powershell
+    $virtualNetwork | Set-AzVirtualNetwork
+    ```
+
+1. Prepare the steps for SFMC BYOVNET as per [Bring your own virtual network](https://learn.microsoft.com/azure/service-fabric/how-to-managed-cluster-networking#bring-your-own-virtual-network)
+
+    - Get the service Id from your subscription for Service Fabric Resource Provider application:
+
+      ```powershell
+      $sfrpPrincipals = @(Get-AzADServicePrincipal -DisplayName 'Azure Service Fabric Resource Provider')
+      ```
+
+    - Obtain the SubnetId from the existing VNet:
+
+      ```powershell
+      $sfmcSubnetId = ((Get-AzVirtualNetwork -Name $vnet.name -ResourceGroupName $resourceGroupName).Subnets | Where Name -eq $sfmcSubnet.Name | Select Id).Id
+      ```
+
+    - Run the following PowerShell command using the principal ID from previous steps, and assignment scope Id obtained above:
+
+      ```powershell
+      foreach($sfrpPrincipal in $sfrpPrincipals) {
+        New-AzRoleAssignment -PrincipalId $sfrpPrincipal.Id -RoleDefinitionName 'Network Contributor' -Scope $sfmcSubnetId
+      }
+      ```
+
+1. Create a Public IP Address (for APIM)
+
+    ```powershell
+    $domainNameLabel = 'apimip'
+    $ip = @{
+        Name = 'apimip'
+        ResourceGroupName = $resourceGroupName
+        Location = $location
+        Sku = 'Standard'
+        AllocationMethod = 'Static'
+        IpAddressVersion = 'IPv4'
+        DomainNameLabel = $domainNameLabel
+    }
+
+    New-AzPublicIpAddress @ip
+    ```
+
+1. Create API Management Service with External VNET integration (It takes around 1 hour)
+
+    ```powershell
+    $apimSubnetId = ((Get-AzVirtualNetwork -Name $vnet.name -ResourceGroupName $resourceGroupName).Subnets | Where Name -eq $apimSubnet.Name | Select Id).Id
+    $apimNetwork = New-AzApiManagementVirtualNetwork -SubnetResourceId $apimSubnetId
+    $publicIpAddressId = (Get-AzPublicIpAddress -Name $ip.name -ResourceGroupName $resourceGroupName | Select Id).Id
+    $apimName = 'myApimCloud'
+    $adminEmail = 'admin@contoso.com'
+    $organization = 'contoso'
+
+    New-AzApiManagement -ResourceGroupName $resourceGroupName `
+      -Location $location `
+      -Name $apimName `
+      -Organization $organization `
+      -AdminEmail $adminEmail `
+      -VirtualNetwork $apimNetwork `
+      -VpnType 'External' `
+      -Sku 'Developer' `
+      -PublicIpAddressId $publicIpAddressId
+    ```
+
+1. Create the SFMC within the VNET previously created.
+    > Example existing vnet managed cluster ARM template below: [SFMC ARM deployment template](#sfmc-arm-deployment-template).
```

Comment:
VNET
managed cluster ARM template?

Reply:
added more description and updated link name


1. Create the Service Fabric Managed Cluster within the existing VNET previously created above.
    > An example existing VNET Service Fabric Managed Cluster ARM template is provided below: [\sfmc-template.json](#sfmc-arm-deployment-template).